crates/ruff_linter/src/rules/flake8_simplify/rules/ast_bool_op.rs

```diff
@@ -429,17 +442,14 @@ pub(crate) fn duplicate_isinstance_call(checker: &mut Checker, expr: &Expr) {
                     let call = node2.into();
 
                     // Generate the combined `BoolOp`.
+                    let before = values.iter().take(indices[0]).map(Clone::clone);
+                    let after = values
+                        .iter()
+                        .skip(indices[indices.len() - 1] + 1)
+                        .map(Clone::clone);
```

Comment:
(I know it was already written this way, but I think `.map(Clone::clone)` can be replaced with `.cloned()`.)